Issue When Opening Chat in 1.16.4

When you hit any button that opens chat; by default `T` or `/` the game crashes.

Error: 
[crash-2020-11-04_17.44.26-client.txt](https://github.com/zeroeightysix/KAMI/files/5491442/crash-2020-11-04_17.44.26-client.txt)

> Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow field slashRequired was not located in the target class net.minecraft.class_4717. Using refmap KAMI-refmap.json

KAMI's command system Mixin for command previews fails on 1.16.4 due to the class being refactored and the field just missing.

--- a/src/main/java/me/zeroeightsix/kami/mixin/client/MixinCommandSuggestor.java
+++ b/src/main/java/me/zeroeightsix/kami/mixin/client/MixinCommandSuggestor.java
@@ -25,7 +25,7 @@ public abstract class MixinCommandSuggestor {
 
     @Shadow
     @Final
-    private boolean slashRequired;
+    private boolean slashOptional;
 
     @Shadow
     private ParseResults<CommandSource> parse;
@@ -48,7 +48,7 @@ public abstract class MixinCommandSuggestor {
             cancellable = true,
             locals = LocalCapture.CAPTURE_FAILHARD)
     public void refresh(CallbackInfo ci, String string, StringReader stringReader) {
-        if (slashRequired) return; // Command block
+        if (!slashOptional) return; // Command block
 
         int i;
         if (stringReader.canRead() && stringReader.peek() == Settings.INSTANCE.getCommandPrefix()) {

